packages/react-native-bottom-tabs/android/src/newarch/RCTTabViewManager.kt

```diff
@@ -43,6 +43,11 @@ class RCTTabViewManager(context: ReactApplicationContext) :
 
   }
 
+  override fun onDropViewInstance(view: ReactBottomNavigationView) {
```

Comment:
We also need this function in `src/oldarch/RCTTabViewManager.kt`

Reply:
Oh yeah you are right. Forgot about that. Will fix